index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Your index.html, style.css, and test.md files should have been added in separate commits according to the project instructions, please re-add them again following the project requirement instructions so we can comply with them. This makes it easier to review certain files and single changes in your commits instead of trying to look for all the changes that you did in 1 big commit, it is cleaner and looks better in my opinion 👍